app/helpers/consignment_inquiries_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ConsignmentInquiriesHelper
+end
```

Comment:
**suggestion:** I guess, we could remove this file if it's not needed.

Reply:
Thanks. I will remove it.